extensions/objects/models/objects.py

```diff
@@ -824,3 +824,23 @@ class ListOfSetsOfHtmlStrings(BaseObject):
         'type': 'list',
         'items': SetOfHtmlString.SCHEMA,
     }
+
+
+class DragAndDropHtmlString(BaseObject):
+    """A drag and drop Html string class."""
+
+    description = 'A drag and drop Html string.'
```

Comment:
A specific drag and drop item from collection of drag and drop items?

Reply:
Done